dfttk/atat/install/README.md

```diff
@@ -1,27 +1,29 @@
 # Installation
-- Download the ATAT source code from https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/.
+1. Download the ATAT source code from https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/. If ATAT is already installed skip to step 3.
 
-- Unzip the package
+2. Unzip the package
 
         tar -xvf atat*.tar.gz
 
-- Copy icamag.c++, fxvector.h, and makefile from this folder into atat/src. Then navigate to the atat/src folder and type:
+3. Remove the makefile located atat/src if it exists.
+
+4. Copy icamag.c++, fxvector.h, and makefile from this folder into atat/src. Then navigate to the atat/src folder and type:
 
```

Comment:
The makefile and fxvector.h files always exists. Maybe something like "replace those files".

Reply:
> The makefile and fxvector.h files always exists. Maybe something like "replace those files".

changed. if you prefer a different wording, please feel free to press `.` to open the editor and change the wording.